vmware_aria_operations_integration_sdk/describe.py

```diff
@@ -368,24 +380,48 @@ def add_identifier(parent: Element, identifier_json: Dict, names: _Names) -> Ele
         },
         nsmap=ns_map,
     )
-    add_enum_values(identifier_xml, identifier_json)
+    add_enum_values(identifier_xml, identifier_json, schema_version, names)
     return identifier_xml
 
 
-def add_enum_values(parent: Element, identifier_json: Dict) -> None:
+def add_enum_values(
+    parent: Element, identifier_json: Dict, schema_version: str, names: _Names
+) -> None:
     if "enum_values" in identifier_json:
-        for value in identifier_json["enum_values"]:
-            SubElement(
-                parent,
-                "enum",
-                attrib={
-                    "value": str(value),
-                    "default": str(
-                        value == identifier_json.get("default", False)
-                    ).lower(),
-                },
-                nsmap=ns_map,
-            )
+        if schema_version == "1.0.0":
```

Comment:
If we have to start supporting a bunch of versions, it might be easier to just use integers - otherwise we'll have to parse this version string in order to do logical comparisons.